Ms teams ring user in authcode (#39392)
demisto/content: Packs/MicrosoftTeams/ReleaseNotes/1_5_22.md

#### Integrations

##### Microsoft Teams

> - ***ms-teams-ring-user*** is now enabled while using the **Authorization Code** authentication type.

Reviewer: ```suggestion
Updated the ***ms-teams-ring-user*** command to be enabled while using the **Authorization Code** authentication type.
```
